src/forms/select/Select.tsx

```diff
@@ -14,7 +14,7 @@ import {Icon} from "../../icon";
 import {classNames} from "../../utils";
 
 type SelectOption = {
-    value?: string;
+    value?: any;
```

Comment:
Why are we switching from String to any ?  Any opens door for more problems as strict checking is close to disabled in this case.

Reply:
@maheshkulkarni4  `Value` prop has `any` type in Select component and here it was string. So I changed it to any. With this we can create Select of type number , string or any other options. Let me know your thoughts on this.